daemon/api_test.go

```diff
@@ -1384,17 +1302,9 @@ func (s *apiSuite) TestLoginUserTwoFactorRequiredError(c *check.C) {
 }
 
 func (s *apiSuite) TestLoginUserTwoFactorFailedError(c *check.C) {
-	serializedMacaroon, err := s.makeStoreMacaroon()
-	c.Assert(err, check.IsNil)
-	responseData, err := s.makeStoreMacaroonResponse(serializedMacaroon)
-	c.Assert(err, check.IsNil)
-	mockDeveloperAPIServer := s.makeDeveloperAPIServer(200, responseData)
-	defer mockDeveloperAPIServer.Close()
-
-	discharge := `{"code": "TWOFACTOR_FAILURE"}`
```

Comment:
The low level bits are tested in auth_store.go:TestDischargeAuthCaveatFails2fa()